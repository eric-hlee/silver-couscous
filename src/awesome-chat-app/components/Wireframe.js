import React from 'react'
import SplitPane from './SplitPane'
import AddChat from '../containers/AddChat'
import History from '../containers/ChatHistory'
import Chat from '../containers/RenderChat'

class Wireframe extends React.Component {

  render() {

    const style_wireframe = {
      height: '80vh',
    }

    return (
      <div className="Wireframe" style={style_wireframe}>
        <AddChat title="An awesome chat app using Redux"></AddChat>
        <SplitPane
            left={
              <History />
            }
            right={
              <Chat />
            }
          />
      </div>
    );
  }
}

export default Wireframe