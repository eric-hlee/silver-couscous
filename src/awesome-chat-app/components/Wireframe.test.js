import React from 'react'
import { shallow } from 'enzyme'
import Wireframe from './Wireframe'

describe("Wireframe component", () => {
  test("it matches the snapshot", () => {
    const wrapper = shallow(<Wireframe />)
    expect(wrapper).toMatchSnapshot()
  });
});