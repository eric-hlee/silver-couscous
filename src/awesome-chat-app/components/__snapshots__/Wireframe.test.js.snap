// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wireframe component it matches the snapshot 1`] = `
<div
  className="Wireframe"
  style={
    Object {
      "height": "100%",
      "minHeight": "100%",
      "overflow": "hidden",
    }
  }
>
  <Connect(Header)
    title="An awesome chat app using Redux"
  />
  <SplitPane
    left={<UNDEFINED />}
    right={<UNDEFINED />}
  />
</div>
`;
